README.md

```diff
@@ -12,5 +12,7 @@ Fun Facts
   * [VIM Fundamentals](https://frontendmasters.com/courses/vim-fundamentals/)
   * [Developer Productivity](https://frontendmasters.com/courses/developer-productivity/)
   * [Rust and Go](https://frontendmasters.com/courses/typescript-go-rust)
```

Comment:
^-- this one is a duplicate of the one below

Reply:
should be good now! 🙃